soc/arm/realtek_bee/rtl87x2g/soc.c

```diff
@@ -88,7 +92,8 @@ static int rtk_platform_init(void)
 
     //os_pm_init();//power manager porting has not been realized yet.
 
-    secure_os_func_ptr_init();
```

Comment:
Please confirm this function. It's designed for non-secure callback init.

Reply:
I plan to first workaround this issue in Zephyr.
After the sys-patch updates the secure_os_func_ptr_init to differentiate between different OSes, I will remove the workaround here.